google/cloud/spanner/sql_partition_test.cc

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/spanner/sql_partition.h"
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+class SqlPartitionTester {
+ public:
+  SqlPartitionTester() = default;
+  SqlPartitionTester(SqlPartition partition)
+      : partition_(std::move(partition)) {}
+  SqlStatement const& sql_statement() const {
+    return partition_.sql_statement();
+  }
+  std::string const& partition_token() const {
+    return partition_.partition_token();
+  }
+  std::string const& session_id() const { return partition_.session_id(); }
+  std::string const& transaction_id() const {
+    return partition_.transaction_id();
+  }
+  SqlPartition partition() const { return partition_; }
+
+ private:
+  SqlPartition partition_;
+};
+
+class SqlPartitionTest : public ::testing::Test {
```

Comment:
I don't think a test fixture is needed in this code, and I think the code below would be simpler without it. See also https://abseil.io/tips/122

The TL;DR is: `TEST(Foo, Blah)` is generally simpler and therefore generally preferred over `TEST_F(Foo, Blah)`.

Reply:
Done.